Dockerfiles/agent/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM debian:jessie
+FROM debian:stable-slim
```

Comment:
let's pin the version here - slim is 👍 and we can take this opportunity to upgrade to stretch